docs/reference/release-notes/highlights-7.0.0.asciidoc

```diff
@@ -298,6 +298,25 @@ https://www.elastic.co/blog/faster-retrieval-of-top-hits-in-elasticsearch-with-b
 WAND: Faster Retrieval of Top Hits in Elasticsearch].
 //end::notable-highlights[]
 
+//tag::notable-highlights[]
+[float]
+=== Faster Boosting with Feature Fields
+
+The Query DSL provides a multitude of ways to boost or modulate how documents are
+scored.  But a common scenario is boosting an entire document based on some kind
+of static criteria recorded on the document itself.  A common example is a pre-computed
+scoring factor (PageRank-esque metric, global popularity score, etc), or 
+metadata about the data itself.
+
+Elasticsearch now exposes a new `feature` field type, which can be used to index
+a numeric value dedicated to scoring.  At query time, a `feature` can make use of this field
+to boost the relevance of the entire document.  But unlike using query boosts or
+a `function_score` , this new field and query can efficiently skip documents
+that no longer have competitive scores.  This speedup is enabled due to the new
+Block-max WAND support in Lucene, and can lead to astonishingly large
+speedups because large stretches of the index can now be skipped entirely.
+
```

Comment:
For this to be re-used successfully, there needs to be an end tag too:
```suggestion
//end::notable-highlights[]
```

Reply:
:man_facepalming: oops :)